src/bun.js/bindings/bindings.zig

```diff
@@ -5596,6 +5621,18 @@ pub const JSArray = struct {
     pub fn from(globalThis: *JSGlobalObject, arguments: []const JSC.JSValue) JSValue {
         return JSC.JSValue.c(JSC.C.JSObjectMakeArray(globalThis, arguments.len, @as(?[*]const JSC.C.JSObjectRef, @ptrCast(arguments.ptr)), null));
     }
+
+    /// Must be called with a tuple of JSValue
+    pub inline fn createComptime(
+        global: *JSGlobalObject,
+        items: anytype,
+    ) JSValue {
+        const a = JSValue.createEmptyArray(global, comptime items.len);
+        inline for (items, 0..) |item, i| {
+            a.putIndex(global, i, item);
```

Comment:
would putDirectIndex be better?

Reply:
i dont think we have a binding for it, i might be wrong.